src/gui/AstroCalcDialog.hpp

```diff
@@ -433,6 +433,50 @@ private slots:
 
 	//! Remember to redraw active plot when dialog becomes visible
 	bool graphPlotNeedsRefresh;
+
+	enum WUTCategory {
```

Comment:
enums should follow the Qt conventions. i.e. CamelCase with First letter capitalization for both enum type and enum values.

Reply:
This is a deliberate choice: see https://github.com/Stellarium/stellarium/pull/1355#issuecomment-723513125

Reply:
Please follow coding style

Reply:
compare this

(screenshot of https://github.com/axd1967/stellarium/blob/9d0c9c4585006d500e38ffea766b9eb5752af016/src/gui/AstroCalcDialog.hpp#L437-L456)

![image](https://user-images.githubusercontent.com/3529789/98469493-4e996a00-21e0-11eb-8491-361f5ab1185e.png)

with these:

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/gui/SearchDialog.hpp#L111-L120

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/StelFileMgr.hpp#L46-L53

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/StelLocaleMgr.hpp#L121-L129

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/modules/GridLinesMgr.cpp#L75-L90

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/modules/Landscape.hpp#L177-L185

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/modules/NomenclatureItem.hpp#L46-L53

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/modules/Planet.hpp#L106-L115

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/src/core/StelCore.hpp#L73-L81

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/Satellites/src/Satellite.hpp#L51-L62

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/Satellites/src/Satellites.hpp#L168-L175

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/RemoteSync/src/SyncClient.hpp#L40-L51

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/ArchaeoLines/src/ArchaeoLines.hpp#L93-L118

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/NavStars/src/NavStars.hpp#L74-L80

https://github.com/Stellarium/stellarium/blob/29b061ea29ba0479326bec824138f17d5df55e9f/plugins/Scenery3d/src/S3DEnum.hpp#L46-L60


agreed, 

1. there are many different ways, and 
2. a coding style attempts to harmonise this

but CamelCase is 
- more difficult to read (the underscore inserts "brea'd'ing space") and also 
- difficult in the presence of names (Oort, Herschel, Algol, ...) and abbreviations (NGC, IC)

here would be the result:

![image](https://user-images.githubusercontent.com/3529789/98469915-9d480380-21e2-11eb-9f4e-9f2aa6d670f0.png)

I think we can agree that this follows the coding style but is unpalatable... The underscore and caps really help in readability.

Reply:
This is bad reasons for introducing another incompatible coding style...

Reply:
> This is bad reasons for introducing another incompatible coding style...

yes, comparing with other bad examples ;-) is indeed a bad reason... but it will take too much time to change, and the result is worse. I think this is actually a good example why the coding style is not perfect; also, a coding style is a guideline; here, I present solid arguments why the style is not ideal in this case...

Reply:
If you do not like some tradition or law then this is not reason do not follow it

Reply:
Plus you new item should be at end of list to backward compatibility of saving options

Reply:
Nice addition. However, I agree with Alexander that existing coding style guidelines should be followed. If this was not adhered to elsewhere for no good reason (yes, blame me as well...), we should fix that there.